ui/components/DataTable/TableView/SearchedNamespacesModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { SearchedNamespaces } from "../../../lib/types";
+import { IconButton } from "../../Button";
+import Icon, { IconType } from "../../Icon";
+import InfoModal from "../../InfoModal";
+
+interface Props {
+  searchedNamespaces: SearchedNamespaces;
+}
+
+const SearchedNamespacesModal = ({ searchedNamespaces }: Props) => {
```

Comment:
Not understanding where this is used. Where in the UI does this modal appear?

Reply:
Used in `AutomationsTable` and `SourcesTable`